src/audio_excl.rs

```diff
@@ -1,13 +1,12 @@
-use rand;
-use rand::seq::IteratorRandom;
 use std::collections::HashMap;
-use std::fs;
-use std::io;
 use std::path::{Path, PathBuf};
-use std::time::Duration;
-use std::time::SystemTime;
+use std::time::{Duration, SystemTime};
+use std::{fs, io};
+
+use rand::{self, seq::SliceRandom};
 use uuid::Uuid;
```

Comment:
Regorganize imports -- `std` imports first, then third-party crates, then local imports. Also, generally use one line per module (e.g. `use std::time::{Duration, SystemTime};` rather than two separate imports).